content/administration/upgrade.rst

```diff
@@ -156,6 +156,19 @@ project <https://odoo.sh/project>`_.
 
          $ python <(curl -s https://upgrade.odoo.com/upgrade) test -d <your db name> -t <target version>
 
+      .. note::
+         This command has some requirements on the environment it runs in:
+
+         - Some external commands that must be provided by the operating system, normally found in
+           any Linux distribution (including WSL). An error will be displayed if one or several of
+           them are missing.
+         - The system user that executes the command needs to be configured with access to the
+           database. Please refer to the PostgreSQL documentation for this requirement.
```

Comment:
You could add a link under _PostgreSQL documentation_. Either to the very general: https://www.postgresql.org/docs/ or Odoo's documentation (?)
https://www.odoo.com/documentation/15.0/administration/on_premise/deploy.html#postgresql

Reply:
good idea, added the relevant links.